service_worker.js

```diff
@@ -234,10 +235,13 @@ function wait(ms) {
  * @param {number} tabId
  * @return {Promise<boolean>}
  */
-function doesTabExist(tabId) {
-  return new Promise((resolve) => {
-    chrome.tabs.get(tabId, () => resolve(!chrome.runtime.lastError));
-  });
+async function doesTabExist(tabId) {
+  try {
+    await chrome.tabs.get(tabId);
+    return true;
+  } catch (_) {
+    return false;
+  }
```

Comment:
Looks like this stopped working since Manifest v3 upgrade but wasn't noticed before, but now want to use this function more so fixed it.